Проверьте ДЗ

Tatyana58 opened the pull request.

Tatyana58 committed:
Beginning
создание класса Author

Tatyana58 committed:
Beginning
создание класса Author
создание класса Book

Tatyana58 committed:
Beginning
создание класса Author
создание класса Book

Tatyana58 committed:
Beginning
создание класса Author
создание класса Book
Вывели в консоль двух авторов двух книг
Изменили год издания
Имя и фамилию Автора пометили в одну переменную
В Классе Book вызывается класс Author

Tatyana58 committed:
Beginning
создание класса Author
создание класса Book
Вывели в консоль двух авторов двух книг
Изменили год издания
Имя и фамилию Автора пометили в одну переменную
В Классе Book вызывается класс Author
Создали класс Print и перенесли метод main в него

Tatyana58 committed:
Beginning
создание класса Author
создание класса Book
Вывели в консоль двух авторов двух книг
Изменили год издания
Имя и фамилию Автора пометили в одну переменную
В Классе Book вызывается класс Author
Создали класс Print и перенесли метод main в него
Оптимизировали вывод на печать

Tatyana58 committed:
Beginning
создание класса Author
создание класса Book
Вывели в консоль двух авторов двух книг
Изменили год издания
Имя и фамилию Автора пометили в одну переменную
В Классе Book вызывается класс Author
Создали класс Print и перенесли метод main в него
Оптимизировали вывод на печать

Tatyana58 committed:
Beginning
создание класса Author
создание класса Book
Вывели в консоль двух авторов двух книг
Изменили год издания
Имя и фамилию Автора пометили в одну переменную
В Классе Book вызывается класс Author
Создали класс Print и перенесли метод main в него
Оптимизировали вывод на печать
Задание выполнено

Tatyana58 committed:
Beginning
создание класса Author
создание класса Book
Вывели в консоль двух авторов двух книг
Изменили год издания
Имя и фамилию Автора пометили в одну переменную
В Классе Book вызывается класс Author
Создали класс Print и перенесли метод main в него
Оптимизировали вывод на печать
Задание выполнено

OlgaLugacheva commented on src/Print.java:
```diff
@@ -0,0 +1,18 @@
+public class Print {
+    public static void main(String[] args){
+        Author author1 = new Author("Николай", "Прохоренок");
+        Book bookName1 = new Book(new Author(author1.getName(), author1.getSurname()), "Основы Java", 2022);
```
мы же уже создали объект Author, его не нужно создавать еще раз - нужно просто его передать в конструктор Book

....  = new Book (author1, ........)

OlgaLugacheva commented on src/Print.java:
```diff
@@ -0,0 +1,18 @@
+        Author author1 = new Author("Николай", "Прохоренок");
+        Book bookName1 = new Book(new Author(author1.getName(), author1.getSurname()), "Основы Java", 2022);
+        Author author2 = new Author("Анджел", "Леонард");
+        Book bookName2 = new Book(new Author(author2.getName(), author2.getSurname()), "Java Решение практических задач", 2021);
```
здесь аналогично

Tatyana58 committed:
Beginning
создание класса Author
создание класса Book
Вывели в консоль двух авторов двух книг
Изменили год издания
Имя и фамилию Автора пометили в одну переменную
В Классе Book вызывается класс Author
Создали класс Print и перенесли метод main в него
Оптимизировали вывод на печать
Задание выполнено
Исправила ДЗ
